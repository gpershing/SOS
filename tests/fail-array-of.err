Fatal error: exception Failure("First operand of of operator must be an int")
