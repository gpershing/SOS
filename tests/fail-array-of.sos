struct point = {x: float, y: float}
p1: point = {1.2, 2.3}

a: array float = p1 of [0.0]
