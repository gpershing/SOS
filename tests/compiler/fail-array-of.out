First operand of of operator must be an int
